### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by BeastMasterGrinder** [openvinotoolkit/openvino#23783] [Google Summer of Code]
As an aspiring computer science student deeply fascinated by AI and machine learning, I have been actively engaging in projects that leverage my programming skills to solve complex problems. My journey through machine learning and deep learning coursework has equipped me with a solid foundation in these domains, complemented by hands-on project experience in Python and C++.

Super stoked to join you guys and get my first open source experience!

**Comment by theprashasst** [comment]
Hello everyone,

I'm Prashasst, a Computer Science undergrad specializing in Artificial Intelligence. I've gained substantial theoretical knowledge in Machine Learning and Deep Learning and a solid understanding of Python. Now, I'm eager to gain practical experience in this field. I would love to kickstart my Open Source journey under your mentorship.

Thank you! ✨
